client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -1798,11 +1799,10 @@
             controller: AuthoringContainerController,
             controllerAs: 'authoring',
             templateUrl: 'scripts/superdesk-authoring/views/authoring-container.html',
-            require: '^sdAuthoringWorkspace',
-            scope: true,
-            link: function(scope, elem, attrs, workspaceCtrl) {
-                scope.$watch(workspaceCtrl.getItem, function(item) {
-                    scope.authoring.edit(item, true);
+            scope: {},
+            link: function(scope) {
+                scope.$watch(authoringWorkspace.getItem, function(item) {
+                    scope.authoring.edit(item, !authoringWorkspace.readonly);
```

Comment:
We might want do skip this call when `$watcher` is invoked for the first time (due to initialization) and nothing has actually changed yet?